gps/turn/index.html

```diff
@@ -31,7 +31,7 @@ <h2 class="text-left">{% include fa_icon.html information=true class="svg-inline
 <p>Eighteen <abbr title="National Geodetic Survey">NGS</abbr> Continually Operating Reference Stations (CORS) were
   selected along the perimeter and interior of the network and were constrained to the published coordinates to conduct
   the final adjustment of the network coordinates.</p>
-<p>With an activated subscription, set your GPS unit's VRS connection address to <code>168.179.231.9</code> and port
+<p>With an activated subscription, set your GPS unit's VRS connection address to <code>165.239.144.5 Port </code> and port
```

Comment:
looks like you got an extra port in there...
```suggestion
<p>With an activated subscription, set your GPS unit's VRS connection address to <code>165.239.144.5</code> and port
```